docs/reference/setup/bootstrap-checks.asciidoc

```diff
@@ -187,7 +187,7 @@ you must not start Elasticsearch with the serial collector (whether it's
 from the defaults for the JVM that you're using, or you've explicitly
 specified it with `-XX:+UseSerialGC`). Note that the default JVM
 configuration that ships with Elasticsearch configures Elasticsearch to
-use the CMS collector.
+use the CMS collector prior to JDK 14 and to use the G1GC collector with JDK 14+.
```

Comment:
how about:

> use the G1GC collector starting with JDK 14 onwards and defaults to the CMS collection for older JDK versions.

Reply:
I agree that we should reorder this so that G1GC comes first. However, I think we should probably break this into two sentences for better clarity.
> use the G1GC garbage collector with JDK14 and later versions. For earlier JDK versions, the configuration defaults to the CMS collector.

Reply:
Thank you for the review @jrodewig and @dliappis. I updated the doc according to your comments.